guide/13-managing-arcgis-applications/enterprise-sites-guide.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<ul><li>remove the word <em>guide </em>from the url in the hyperlink for Initiatives to reference the name of the ipynb file not the branch name:  (../hub-premium-guide)</li><li>remove the phrase "...as demonstrated here." It's unclear as to where <em>here </em>is since it is not a hyperlink</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>